.circleci/config.yml

```diff
@@ -117,15 +209,11 @@ workflows:
       - build-push-deploy:
           app:                        federalist-build-container<< matrix.extension >>-staging
           space:                      staging
-          cf-username:                CF_USERNAME_STAGING
-          cf-password:                CF_PASSWORD_STAGING
           account-url:                AWS_ECR_ACCOUNT_URL_STAGING
           aws-access-key-id:          AWS_ECR_WRITE_KEY_STAGING
           aws-secret-access-key:      AWS_ECR_WRITE_SECRET_STAGING
           region:                     AWS_ECR_REGION_STAGING
-          repo:                       $AWS_ECR_REPO_STAGING
-          aws-read-access-key-id:     AWS_ECR_READ_KEY_STAGING
-          aws-read-secret-access-key: AWS_ECR_READ_SECRET_STAGING
+          repo:                       federalist-staging-ecr
```

Comment:
is this correct?

Reply:
see https://github.com/18F/federalist-infra/pull/22 for reasoning. that name is correct.